core/src/main/kotlin/no/nav/amt/tiltak/core/port/NavAnsattService.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.amt.tiltak.core.port
+
+import no.nav.amt.tiltak.core.domain.navansatt.NavAnsatt
+
+interface NavAnsattService {
+	// TODO kan (og burde?) denne slås sammen med tiltaks-arrangør sin tilgangservice? InnloggetBrukerService kanskje?
```

Comment:
Tilgangskontrollen for tiltaksarrangor har blitt flyttet til en egen modul i https://github.com/navikt/amt-tiltak/pull/155